azkaban-db/src/main/sql/create.ramp_dependency.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE ramp_dependency (
-    dependency VARCHAR(45) NOT NULL,
-    defaultValue VARCHAR (500),
+    dependency VARCHAR(200) NOT NULL,
+    defaultValue VARCHAR (1000),
```

Comment:
Just curious, is there a retention policy setup at DB or app level for these tables?

Reply:
what do you mean? The constraint check is on DB only. We don't setup the limitation upon the app level.

Reply:
Are the records of these tables deleted after some time, for example after 6 months?

Reply:
yes. From @kxu1026:

> data will be removed after even ramping cycle.
> so, no accumulated items will remain even after couple of weeks.